exonum_client/client.py

```diff
@@ -100,7 +101,12 @@ def run(self) -> None:
 
     def _event_processing(self) -> None:
         while self._is_running:
-            data = self._ws_client.recv()
+            data = None
+            try:
+                data = self._ws_client.recv()
+            except Exception as err:
```

Comment:
Catching `Exception` is bad idea, unless you are catching it just to report and shutdown gracefully.

Reply:
I've changed the order of socket closure and thread termination, since it was causing a blocking behaviour. But now the cal of `recv()` method, mentioned above, can cause an Exception if the `stop()` method of a `Subscriber` instance is called.

Reply:
And there are no other solutions for this?

Reply:
I'll think about a more elegant solution. But any suggestions are welcome )

Reply:
I removed the 'try' block. It will still raise an error with message "OSError: [Errno 9] Bad file descriptor" but since it occurs in a thread, it won't affect the rest of the program.